WordPress/Classes/Models/WPAccount+RestApi.swift

```diff
@@ -13,12 +17,15 @@ extension WPAccount {
         if let api = _private_wordPressComRestApi {
             return api
         }
+
         guard let authToken, !authToken.isEmpty else {
-            DispatchQueue.main.async {
```

Comment:
I don't think we should delete this dispatch async call. We need to either post the notification from the main thread or respond to the notification from the main thread.

Reply:
`wordPressComRestApi` gets accessed from the background threads.